app/components/shares/permission_button_component.rb

```diff
@@ -80,5 +80,21 @@ def form_inputs(role_id)
         inputs << { name: "filters", value: params[:filters] } if params[:filters]
       end
     end
+
+    def tooltip_message
+      return if strategy.manageable?
+
+      I18n.t("sharing.denied", entities: strategy.entity.class.model_name.human(count: 2))
+    end
+
+    def tooltip_wrapper_classes
+      return [] if strategy.manageable?
+
+      ["tooltip--left"]
+    end
+
+    def editable?
+      strategy.manageable? && share.principal != User.current
```

Comment:
User can't edit own share?

Reply:
No, you cannot edit your own share. This was already the case in the Workpackage implementation and got lost when generalizing